src/components/Booking/Booking.jsx

```diff
@@ -0,0 +1,31 @@
+import dayjs from "dayjs";
+
+import "./Booking.scss"
+
+const Booking = (props) => {
+  const {
+    title,
+    firstName,
+    surname,
+    email,
+    roomId,
+    checkInDate,
+    checkOutDate,
+  } = props.booking;
+  const stayNights = dayjs(checkOutDate).diff(dayjs(checkInDate), "d");
+
+  return (
+    <tr data-testid="booking-component">
+      <td className="customer-name">
+        {title} {firstName} {surname}
+      </td>
+      <td className="customer-email">{email}</td>
+      <td className="customer-room">{roomId}</td>
+      <td className="customer-checkin">{checkInDate}</td>
+      <td className="customer-checkout">{checkOutDate}</td>
+      <td className="customer-stay">{stayNights} {stayNights > 1 ? 'nights' : 'night'}</td>
+    </tr>
+  );
+};
+
+export default Booking;
```

Comment:
good work!
Again I would suggest some of the code can be written in one line, but if its easier for you like this then go ahead.

Reply:
Hi Hadika,
Could you provide an example? I'd like to see if changing to this approach would make it easier for everyone to understand.

Reply:
So in this specific code, you could write const for props.booking like this: 
const { title, firstName, surname, email, roomId, checkInDate, checkOutDate } = props.booking;
For me this is easier to read and takes up much less code space.